nss-sapphire-elephants/WorkforceManagement
# Cw training list all and create

Completion of Issue Tickets 9 and 11
LIST ALL TRAININGS
Given a user wants to view all training programs
When the user clicks the Training Programs item in the navigation bar
Then the user will see a list of all training programs that have not taken place yet

CREATE A TRAINING
Given the user is viewing all training programs
When the user clicks the Create New link
Then the user should be presented with a form in which the following information can be entered:
Name, Title, Start day, End day. Maximum number of attendees

DELETE A TRAINING
Given user wants to remove a training program
When the user clicks on a training program
And then clicks on the delete button
Then the training program should be deleted unless the training program has already taken place

Testing Instructions for Change Made:
1. git fetch --all
2. git checkout cw-training-ListAll-AndCreate
3. run

Make sure there is a list of trainings with a name, a start date, an end date a maximum number of employees for future trainings
Create a new Training and make sure the User can instantly see their newly added Training listed.
Delete a Training and make sure the User can NOT see their newly deleted Training.

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings
 I have added test instructions that prove my fix is effective or that my feature works
